jmx-metrics/src/integrationTest/resources/wildfly/start.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+/opt/jboss/wildfly/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0 & \
+
+/opt/jboss/wildfly/bin/add-user.sh user password --silent & \
+
+java -cp /app/OpenTelemetryJava.jar:/opt/jboss/wildfly/bin/client/jboss-client.jar \
+  -Dotel.jmx.username=user -Dotel.jmx.password=password \
```

Comment:
believe these are unnecessary

Reply:
I may be mistaken, but I believe I need those. When the container starts up, I need to start the WildFly process, add a user (since it seems to be a requirement with this JMX implementation), and start the metric scraper. 

I tried to combine these all in a command when creating the generic container, but I discovered it's fairly complicated to execute multiple commands. From what I've seen, it's easier to create a startup script and execute that in your command when creating the generic container. Also, it turns out that the server startup is blocking, so that somewhat complicates matters further, as we need to execute that in the background.

Reply:
They were unnecessary before being removed from the properties file in your later change.